.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: haskell
+
+ghc:
```

Comment:
Ha, I thought I was commenting on Carnival. Yeah, I think what you're doing makes sense here.